PinvokeAnalyzer_UWPApis.txt

```diff
@@ -0,0 +1,2298 @@
+bcrypt.dll!BCryptCloseAlgorithmProvider
```

Comment:
This is an updated version of API list that used to come with build tools. APISet names have been mapped to corresponding Win32 DLL names, while non-APISet imports from the original list are copied as-is.